### PR TITLE
Adding En dash rule

### PR DESCRIPTION
**Pull request by aireilly**
Fixes https://github.com/redhat-documentation/vale-at-red-hat/issues/698

> Generally, use an en dash (–) without spaces on each side to indicate a range of numbers. If you list numbers along with other characters, unless the context determines a different order of the characters, list numeric characters first, followed by alphabetic characters (uppercase and then lowercase), followed by other characters in ASCII order.

### CODE REVIEW

File: modules/reference-guide/pages/endashes.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+:navtitle: Endashes
+:keywords: reference, rule, endashes
+
+= En Dashes
```
Comment (apinnick) [inline modules/reference-guide/pages/endashes.adoc:4]: Small nit. I have not checked the other "pages". Do they use title case or sentence case?

**Review by apinnick — APPROVED** [review]
Apart from a query about the "page" title, LGTM.

File: modules/reference-guide/pages/endashes.adoc (hunk 1 above)
Comment (aireilly) [reply modules/reference-guide/pages/endashes.adoc:4]: Cool. Sentence case I guess. thanks!

**Comment by aireilly** [comment]
In retrospect, I am going to close this un-merged. Terms like the below perfectly valid and will get caught in this rule.

```
ip-10-0-0-1.us-east-2.compute.internal
775898c3-ecc2-41a4-b98b-a4cd5ae56fd0
2023-09-22
```

**Pull request closed without merging**